Neos.Fusion/Classes/Core/RuntimeConfiguration.php

```diff
@@ -0,0 +1,235 @@
+<?php
+
+namespace Neos\Fusion\Core;
+
+use Neos\Fusion\Exception;
+use Neos\Flow\Annotations as Flow;
+use Neos\Utility\Arrays;
+
+/**
+ * @Flow\Proxy(false)
+ */
+final class RuntimeConfiguration
+{
+
+    /**
+     * The parsed Fusion configuration
+     *
+     * @var array
+     */
+    private $fusionConfiguration;
+
+    /**
+     * @var array
```

Comment:
```suggestion
     * @var string[]
```

Reply:
Well, `pathCache` is actually an array of arrays indexed by string... Dunno how to type that in PHP ;)